test/syscalls/linux/proc_net.cc

```diff
@@ -54,6 +62,188 @@ TEST(ProcSysNetIpv4Sack, CanReadAndWrite) {
   EXPECT_EQ(buf, to_write);
 }
 
+bool get_snmp_metric(std::string snmp, const char *type, const char *item,
+		     uint64_t *p_val) {
```

Comment:
igudger: It looks like all users of this function expect that the result is true, meaning that false is an error condition. In cases like this, we typically return an ErrorOr style type which includes an error or a value. See CurrentRSS in proc.cc for an example of how to do this.

https://gvisor-review.git.corp.google.com/c/gvisor/+/18283/6/test/syscalls/linux/proc_net.cc#66

Reply:
Done.